app/models/foreman_ansible/ansible_provider.rb

```diff
@@ -62,6 +62,14 @@ def proxy_operation_name
           'ansible-runner'
         end
 
+        def required_proxy_selector_for(template)
+          if template.remote_execution_features.where(:label => 'ansible_run_capsule_upgrade').any?
+            ::InternalProxyProxySelector.new
```

Comment:
Where is this class defined?

Reply:
And of course I forgot to add the new file to the commit. It is defined in the REX PR this one depends on.

https://github.com/theforeman/foreman_remote_execution/pull/490/files#diff-00044ee009a2a6f328c8251ee0dc386d